auto_round/script/mllm.py

```diff
@@ -392,6 +392,12 @@ def tune(args):
 
     if "--truncation" not in sys.argv:
         args.truncation = None
+    
+    if "auto_awq" in args.format:
+        from auto_round.utils import check_awq_gemm_compatibility
+        awq_supported, info = check_awq_gemm_compatibility(model,args.bits,args.group_size, not args.asym, layer_config)
```

Comment:
in this function, only llm part has been checked. You need to check qvision part too